### PR TITLE
fix(core): Add missing ASCII art

### PR DESCRIPTION
**Pull request by tomkerkhove**
Improve Helm notes a bit to make it more end-user friendly and add missing ASCII art:

```
:::^.     .::::^:     :::::::::::::::    .:::::::::.                   .^.
7???~   .^7????~.     7??????????????.   :?????????77!^.              .7?7.
7???~  ^7???7~.       ~!!!!!!!!!!!!!!.   :????!!!!7????7~.           .7???7.
7???~^7????~.                            :????:    :~7???7.         :7?????7.
7???7????!.           ::::::::::::.      :????:      .7???!        :7??77???7.
7????????7:           7???????????~      :????:       :????:      :???7?5????7.
7????!~????^          !77777777777^      :????:       :????:     ^???7?#P7????7.
7???~  ^????~                            :????:      :7???!     ^???7J#@J7?????7.
7???~   :7???!.                          :????:   .:~7???!.    ~???7Y&@#7777????7.
7???~    .7???7:      !!!!!!!!!!!!!!!    :????7!!77????7^     ~??775@@@GJJYJ?????7.
7???~     .!????^     7?????????????7.   :?????????7!~:      !????G@@@@@@@@5??????7:
::::.       :::::     :::::::::::::::    .::::::::..        .::::JGGGB@@@&7:::::::::
                                                                      ?@@#~
                                                                      P@B^
                                                                    :&G:
                                                                    !5.
                                                                    .

Kubernetes Event-driven Autoscaling (KEDA) - Application autoscaling made simple.

Get started by deploying Scaled Objects to your cluster:
    - Information about Scaled Objects : https://keda.sh/docs/latest/concepts/
    - Samples: https://github.com/kedacore/samples

Get information about the deployed ScaledObjects:
  kubectl get scaledobject [--namespace <namespace>]

Get details about a deployed ScaledObject:
  kubectl describe scaledobject <scaled-object-name> [--namespace <namespace>]

Get information about the deployed ScaledObjects:
  kubectl get triggerauthentication [--namespace <namespace>]

Get details about a deployed ScaledObject:
  kubectl describe triggerauthentication <trigger-authentication-name> [--namespace <namespace>]

Get an overview of the Horizontal Pod Autoscalers (HPA) that KEDA is using behind the scenes:
  kubectl get hpa [--all-namespaces] [--namespace <namespace>]

Learn more about KEDA:
- Documentation: https://keda.sh/
- Support: https://keda.sh/support/
- File an issue: https://github.com/kedacore/keda/issues/new/choose
```

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

### CODE REVIEW

**Comment by JorTurFer** [comment]
TBH, I'm not a huge fan of ASCII art because it creates too much noise in CIs. But if @zroubalik agrees, I won't say anything xD

**Comment by tomkerkhove** [comment]
image

But hey, that's why it is configurable =)

File: keda/templates/NOTES.txt [hunk 1]
```diff
@@ -20,8 +37,6 @@ Get details about a deployed ScaledObject:
 Get an overview of the Horizontal Pod Autoscalers (HPA) that KEDA is using behind the scenes:
   kubectl get hpa [--all-namespaces] [--namespace <namespace>]
 
--------------------------------------------------------------------------------------
```
Comment (zroubalik) [inline keda/templates/NOTES.txt:23]: The main reason why I added this is if there are any warning printed, they will be right after the line above, ie there is not a empty line printed. And it is not very clear.

File: keda/templates/NOTES.txt [hunk 2]
```diff
@@ -1,6 +1,23 @@
-=====================================================================================
-KEDA - Kubernetes Event Driven Autoscaling
-=====================================================================================
+:::^.     .::::^:     :::::::::::::::    .:::::::::.                   .^.
```
Comment (zroubalik) [inline keda/templates/NOTES.txt:1]: OH MY 😄 

I with @JorTurFer on this one, but not completely against 😄

Comment (JorTurFer) [reply keda/templates/NOTES.txt:1]: 1 + 0,5 + 0,5 = 2 of 3, so we can merge it xD

File: keda/templates/NOTES.txt (hunk 1 above)
Comment (JorTurFer) [reply keda/templates/NOTES.txt:23]: Should we undo this?

File: keda/templates/NOTES.txt (hunk 2 above)
Comment (tomkerkhove) [reply keda/templates/NOTES.txt:1]: > Looks like we can merge it xD

You confused me =)

Is this a blocker or not? `--set asciiArt=false` disables this though

File: keda/templates/NOTES.txt (hunk 1 above)
Comment (tomkerkhove) [reply keda/templates/NOTES.txt:23]: Yes, but let's only show the line if there is an effective warning then. If not, this is just pointless imo

File: keda/templates/NOTES.txt [hunk 3]
```diff
@@ -32,11 +47,11 @@ WARNING - prometheus.metricServer.serviceMonitor.relabellings is deprecated, ple
 WARNING - prometheus.webhooks.serviceMonitor.relabellings is deprecated, please migrate to prometheus.webhooks.serviceMonitor.relabelings instead.
 {{- end }}
 
-
 {{- if lt .Capabilities.KubeVersion.Minor "25" }}
 WARNING - Running on unsupported Kubernetes version "1.{{.Capabilities.KubeVersion.Minor}}". KEDA 2.11 is supported and tested on Kubernetes "1.25" or higher. See https://keda.sh/docs/2.11/operate/cluster/ for details.
```
Comment (tomkerkhove) [inline keda/templates/NOTES.txt:51]:
```suggestion
-------------------------------------------------------------------------------------
WARNING - Running on unsupported Kubernetes version "1.{{.Capabilities.KubeVersion.Minor}}". KEDA 2.11 is supported and tested on Kubernetes "1.25" or higher. See https://keda.sh/docs/2.11/operate/cluster/ for details.
-------------------------------------------------------------------------------------
```

File: keda/templates/NOTES.txt [hunk 4]
```diff
@@ -32,11 +47,11 @@ WARNING - prometheus.metricServer.serviceMonitor.relabellings is deprecated, ple
 WARNING - prometheus.webhooks.serviceMonitor.relabellings is deprecated, please migrate to prometheus.webhooks.serviceMonitor.relabelings instead.
```
Comment (tomkerkhove) [inline keda/templates/NOTES.txt:47]:
```suggestion
-------------------------------------------------------------------------------------
WARNING - prometheus.webhooks.serviceMonitor.relabellings is deprecated, please migrate to prometheus.webhooks.serviceMonitor.relabelings instead.
-------------------------------------------------------------------------------------
```

File: keda/templates/NOTES.txt [hunk 5]
```diff
@@ -20,8 +37,6 @@ Get details about a deployed ScaledObject:
 Get an overview of the Horizontal Pod Autoscalers (HPA) that KEDA is using behind the scenes:
   kubectl get hpa [--all-namespaces] [--namespace <namespace>]
 
--------------------------------------------------------------------------------------
-
 {{- if .Values.prometheus.operator.serviceMonitor.relabellings}}
 WARNING - prometheus.operator.serviceMonitor.relabellings is deprecated, please migrate to prometheus.operator.serviceMonitor.relabelings instead.
```
Comment (tomkerkhove) [inline keda/templates/NOTES.txt:41]:
```suggestion
-------------------------------------------------------------------------------------
WARNING - prometheus.operator.serviceMonitor.relabellings is deprecated, please migrate to prometheus.operator.serviceMonitor.relabelings instead.
-------------------------------------------------------------------------------------
```

File: keda/templates/NOTES.txt (hunk 1 above)
Comment (tomkerkhove) [reply keda/templates/NOTES.txt:23]: Updated

File: keda/templates/NOTES.txt (hunk 2 above)
Comment (JorTurFer) [reply keda/templates/NOTES.txt:1]: does helm support that? or how does it work?
We can merge it

Comment (tomkerkhove) [reply keda/templates/NOTES.txt:2]: Yes! If I would at least push the commit where I added this :)
https://github.com/kedacore/charts/pull/484/commits/0814f1b5abc511271bb92e0ce62b2fa4f60cd17a

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**